{"name": "test_order_product", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nBacktrace:\n\tOrdinal0 [0x011E6903+2517251]\n\tOrdinal0 [0x0117F8E1+2095329]\n\tOrdinal0 [0x01082848+1058888]\n\tOrdinal0 [0x010AD448+1233992]\n\tOrdinal0 [0x010AD63B+1234491]\n\tOrdinal0 [0x010D7812+1406994]\n\tOrdinal0 [0x010C650A+1336586]\n\tOrdinal0 [0x010D5BBF+1399743]\n\tOrdinal0 [0x010C639B+1336219]\n\tOrdinal0 [0x010A27A7+1189799]\n\tOrdinal0 [0x010A3609+1193481]\n\tGetHandleVerifier [0x01375904+1577972]\n\tGetHandleVerifier [0x01420B97+2279047]\n\tGetHandleVerifier [0x01276D09+534521]\n\tGetHandleVerifier [0x01275DB9+530601]\n\tOrdinal0 [0x01184FF9+2117625]\n\tOrdinal0 [0x011898A8+2136232]\n\tOrdinal0 [0x011899E2+2136546]\n\tOrdinal0 [0x01193541+2176321]\n\tBaseThreadInitThunk [0x7600343D+18]\n\tRtlInitializeExceptionChain [0x772E9812+99]\n\tRtlInitializeExceptionChain [0x772E97E5+54]", "trace": "self = <tests.test_homepage.TestHomePage object at 0x03BEDCD0>\n\n    @allure.severity(allure.severity_level.CRITICAL)\n    def test_order_product(self):\n        \"\"\"Verifying the ability to order a product\"\"\"\n        order = HomePage(self.driver)\n        order.search_by_text(\"Dress\")\n        order.click_search_button()\n>       assert order.check_results_exists(), 'There are no items in results found'\n\ntests\\test_homepage.py:69: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npages\\homepage.py:40: in check_results_exists\n    res_count = self.is_present('xpath', HomePageLocators.X_results_found)\nbase\\seleniumbase.py:29: in is_present\n    return self.__wait.until(ec.presence_of_element_located((self.__get_selenium_by(find_by), locator)))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"f07048b9d3525367333ecda79381e451\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x03BE8A90>\nmessage = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except InvalidSelectorException as e:\n                raise e\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       Backtrace:\nE       \tOrdinal0 [0x011E6903+2517251]\nE       \tOrdinal0 [0x0117F8E1+2095329]\nE       \tOrdinal0 [0x01082848+1058888]\nE       \tOrdinal0 [0x010AD448+1233992]\nE       \tOrdinal0 [0x010AD63B+1234491]\nE       \tOrdinal0 [0x010D7812+1406994]\nE       \tOrdinal0 [0x010C650A+1336586]\nE       \tOrdinal0 [0x010D5BBF+1399743]\nE       \tOrdinal0 [0x010C639B+1336219]\nE       \tOrdinal0 [0x010A27A7+1189799]\nE       \tOrdinal0 [0x010A3609+1193481]\nE       \tGetHandleVerifier [0x01375904+1577972]\nE       \tGetHandleVerifier [0x01420B97+2279047]\nE       \tGetHandleVerifier [0x01276D09+534521]\nE       \tGetHandleVerifier [0x01275DB9+530601]\nE       \tOrdinal0 [0x01184FF9+2117625]\nE       \tOrdinal0 [0x011898A8+2136232]\nE       \tOrdinal0 [0x011899E2+2136546]\nE       \tOrdinal0 [0x01193541+2176321]\nE       \tBaseThreadInitThunk [0x7600343D+18]\nE       \tRtlInitializeExceptionChain [0x772E9812+99]\nE       \tRtlInitializeExceptionChain [0x772E97E5+54]\n\n..\\..\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:89: TimeoutException"}, "description": "Verifying the ability to order a product", "steps": [{"name": "Набор заданного текста в поле поиска", "status": "passed", "parameters": [{"name": "text", "value": "'Dress'"}], "start": 1642165119682, "stop": 1642165119881}, {"name": "Клик по кнопке \"Найти\"", "status": "passed", "start": 1642165119881, "stop": 1642165120145}, {"name": "Проверка: поиск выдал результаты", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nBacktrace:\n\tOrdinal0 [0x011E6903+2517251]\n\tOrdinal0 [0x0117F8E1+2095329]\n\tOrdinal0 [0x01082848+1058888]\n\tOrdinal0 [0x010AD448+1233992]\n\tOrdinal0 [0x010AD63B+1234491]\n\tOrdinal0 [0x010D7812+1406994]\n\tOrdinal0 [0x010C650A+1336586]\n\tOrdinal0 [0x010D5BBF+1399743]\n\tOrdinal0 [0x010C639B+1336219]\n\tOrdinal0 [0x010A27A7+1189799]\n\tOrdinal0 [0x010A3609+1193481]\n\tGetHandleVerifier [0x01375904+1577972]\n\tGetHandleVerifier [0x01420B97+2279047]\n\tGetHandleVerifier [0x01276D09+534521]\n\tGetHandleVerifier [0x01275DB9+530601]\n\tOrdinal0 [0x01184FF9+2117625]\n\tOrdinal0 [0x011898A8+2136232]\n\tOrdinal0 [0x011899E2+2136546]\n\tOrdinal0 [0x01193541+2176321]\n\tBaseThreadInitThunk [0x7600343D+18]\n\tRtlInitializeExceptionChain [0x772E9812+99]\n\tRtlInitializeExceptionChain [0x772E97E5+54]\n\n", "trace": "  File \"C:\\Users\\Artem Kozlov\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\Artem Kozlov\\PycharmProjects\\POM_Selenium_Pytest_Task1\\pages\\homepage.py\", line 40, in check_results_exists\n    res_count = self.is_present('xpath', HomePageLocators.X_results_found)\n  File \"C:\\Users\\Artem Kozlov\\PycharmProjects\\POM_Selenium_Pytest_Task1\\base\\seleniumbase.py\", line 29, in is_present\n    return self.__wait.until(ec.presence_of_element_located((self.__get_selenium_by(find_by), locator)))\n  File \"C:\\Users\\Artem Kozlov\\AppData\\Local\\Programs\\Python\\Python38-32\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 89, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1642165120146, "stop": 1642165130468}], "attachments": [{"name": "Screenshot", "source": "07dac134-4bb6-4f73-9ccf-e9e938852391-attachment.png", "type": "image/png"}], "start": 1642165107194, "stop": 1642165130508, "uuid": "f731996b-54b3-4918-ae9a-bfc7dd7bf7f5", "historyId": "99784ccf2a6379dc9b2e2ed1dce6824d", "testCaseId": "a031fcf4d13fed04f9d232ba23d4bcbc", "fullName": "tests.test_homepage.TestHomePage#test_order_product", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "@pytest.mark.usefixtures('web_driver_init')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_homepage"}, {"name": "subSuite", "value": "TestHomePage"}, {"name": "host", "value": "ArtemKozlov"}, {"name": "thread", "value": "4620-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_homepage"}]}